src/mitol/common/decorators.py

```diff
@@ -7,20 +8,23 @@
 def cache_control_max_age_jitter(*args, **kwargs):
     def _cache_controller(viewfunc):
         @wraps(viewfunc)
-        def _cache_controlled(request, *args, **kwargs):
+        def _cache_controlled(request, *args, **kw):
             # Ensure argument looks like a request.
             if not hasattr(request, "META"):
                 raise TypeError(
                     "cache_control_max_age_jitter didn't receive an HttpRequest. If you are "
                     "decorating a classmethod, be sure to use "
                     "@method_decorator."
                 )
-            response = viewfunc(request, *args, **kwargs)
+            response = viewfunc(request, *args, **kw)
             max_age = get_max_age(response)
             # add random delay upto 5 minutes
-            if max_age:
-                kwargs["max_age"] = max_age + random.randint(1, 600)
-                patch_cache_control(response, **kwargs)
+            
+            if not max_age:
+                # we could set it through the settings.py variable settings.WAGTAIL_IMG_CACHE_AGE
```

Comment:
Could you remove this comment, this code isn't specific to wagtail and I'd want to avoid confusion.